Source/Core/CommandLineOptions.cs

```diff
@@ -1911,6 +1913,13 @@ identifies variables
   /printUnstructured : with /print option, desugars all structured statements
   /printDesugared : with /print option, desugars calls
 
+  /freeVarLambdaLifting : Boogie's lambda lifting transforms the bodies of lambda
+                         expressions into templates with holes. By default, holes
+                         are maximally large subexpressions that do not contain
+                         bound variables (see Section 8 of Boogie 3 paper).
```

Comment:
Leave out the parenthetic remark, since the paper you're referring to is not available.